### PR TITLE
Add a CONTRIBUTING.md so folks know how to engage with this initiative

### PR DESCRIPTION
**Pull request by webchick**
Here is some information that was shared in Slack, captured here for posterity, to help newcomers to the project hit the ground running.

Credit also to @kdruckman who helped answer open questions!

Note: This document uses a pretty informal tone, and if need be I can "tone" it down 🥁  to read more like the rest of the documentation for OpenSSF. However, given this is a community-facing initiative, I thought it might be good to show some "personality" here, since we can be both a fun group to work with and **also** get Very Important Business™ done. :)

### CODE REVIEW

**Comment by funnelfiasco** [comment]
As someone who just discovered this community and is looking to get started, this is a great v1. :ship:

**Comment by Cheukting** [comment]
@webchick this is awesome! I think the tone is landing at the sweet spot and I love it! I have added some small reviews otherwise we can merge!

**Comment by webchick** [comment]
Yay, thanks! 😄 Also I'm not sure if it's the jet lag talking, but I don't seem to see your review comments?

**Review by Cheukting — CHANGES_REQUESTED** [review]

**Comment by Cheukting** [comment]
> Yay, thanks! 😄 Also I'm not sure if it's the jet lag talking, but I don't seem to see your review comments?

@webchick my bad, I forgot to actually submit these reviews so there where pending and hidden >_<

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* Open Source Maintainers, to provide “maintainer experience” feedback on OpenSSF tools and presentations
+
+If you don't see your exact experience listed here, please join anyway! :D
+
+## Ways to Contribute
+### Meet us at… meetings!
+You can talk with other folks interested in this initiative in “real time” during:
+
+* **Monthly Monthly DevRel Community meetings** which happen the second Thursday of the month at 19:30 UTC ([your timezone](https://time.is/1930_in_UTC)) for updates of general goings on and general discussion. Feel free to add a topic to the [agenda](https://docs.google.com/document/d/1qhqQKezqt4R9tfy3gh6MwatG-Tbs7MDPb2_LVzaeMOY/edit) and you can find notes from previous meetings in the [meetings](./meetings) folder.
+* **Weekly Office Hours** which are weekly at 20:30 UTC ([your timezone](https://time.is/2030_in_UTC)) These are generally working sessions, but we hope to expand to activities like reviewing CFP submissions, offer space for folks to practice talks, etc.
```
[the remark this answers is not in the document]
Comment (webchick) [reply CONTRIBUTING.md:44]: D'oh. I apparently got Eastern (what's shown on the [calendar](https://openssf.org/getinvolved/)) and my time zone confused. 🙈

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+If you're new to OpenSSF, you can watch or attend a [Town Hall](https://www.youtube.com/playlist?list=PLVl2hFL_zAh_rQ4oTpM7OKkEm_Y2OWsK8) to get acquainted, as well as join in to any of the other meetings in the calendar at [Get Involved](https://openssf.org/getinvolved/) to get a bit more “in the weeds” with an area that interests you!
```
[the remark this answers is not in the document]
Comment (webchick) [reply CONTRIBUTING.md:46]: OH, rad! I've never seen that before! +1

**Comment by webchick** [comment]
OK, I believe this is now done; back to you! :D

**Review by Cheukting — APPROVED** [review]
Awesome!!!!!

**Comment by Cheukting** [comment]
@webchick when you are ready, feel free to merge :-)

**Comment by webchick** [comment]
Yay thanks!! 😄 @kdruckman requested a review as well, so I’ll hold to see what she has to say. (As well if there are any other comments.)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* Folks with Community Management experience who can help with herding cats
+* Open Source Maintainers, to provide “maintainer experience” feedback on OpenSSF tools and presentations
+
+If you don't see your exact experience listed here, please join anyway! :D
+
+## Ways to Contribute
+### Meet us at… meetings!
+You can talk with other folks interested in this initiative in “real time” during:
+
+* **Monthly Monthly DevRel Community meetings** which happen the second Thursday of the month at 16:30 UTC ([your timezone](https://time.is/1630_in_UTC)) for updates of general goings on and general discussion. Feel free to add a topic to the [agenda](https://docs.google.com/document/d/1qhqQKezqt4R9tfy3gh6MwatG-Tbs7MDPb2_LVzaeMOY/edit) and you can find notes from previous meetings in the [meetings](./meetings) folder.
```
Comment (kdruckman) [inline CONTRIBUTING.md:43]: "Monthly" is repeated.

Comment (kdruckman) [inline CONTRIBUTING.md:43]: "Monthly" is repeated.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+See the [Meetup Organizing Guidelines](https://openssf.org/community/meetup-guidelines/).
+
+### Contribute an OpenSSF Talk / Abstract
+If you have an OpenSSF-related talk you’ve given in the past, and/or on one of OpenSSF’s related initiatives, you can submit a PR to add it to our [talk templates](./talk_templates).
+
+### *Give* an OpenSSF Talk
+If you’re interested in giving a talk about OpenSSF at a conference targeting open source developers and/or maintainers, you can find a list of [sample abstracts](./abstracts) as well as [talk templates](./talk_templates).
+
+If you’d like feedback on your abstract or your talk, please join us during weekly office hours!
```
Comment (kdruckman) [inline CONTRIBUTING.md:64]: Should we link to the meeting calendar? (maybe also for the above meeting references too)
```suggestion
If you’d like feedback on your abstract or your talk, please join us during weekly office hours, which you can find on the [OpenSSF Community Calendar](https://openssf.org/getinvolved/)!
```

**Review by kdruckman — CHANGES_REQUESTED** [review]
This is fabulous @webchick! I only have a minor change and small suggestion.

File: CONTRIBUTING.md (hunk 3 above)
Comment (webchick) [reply CONTRIBUTING.md]: Well I COULD fix that but then are we sure people would know they’re monthly? I feel like maybe I should add another one for EMPHASIS!! 😂

Fixed, thank you!!

**Comment by webchick** [comment]
Yay!! Thank you for the review! I believe now all review comments have been resolved, so back to you and will merge this once @kdruckman has signed off.

**Review by kdruckman — APPROVED** [review]
Looks good to merge!

**Comment by webchick** [comment]
Woohoo! Thanks so much, all! 😄

**Pull request merged**